pkg/util/validation/limits.go

```diff
@@ -474,3 +474,19 @@ func SmallestPositiveNonZeroDurationPerTenant(tenantIDs []string, f func(string)
 	}
 	return *result
 }
+
+// BiggestDurationPerTenant is returning the maximum duration
+// per tenant. Without tenants given it will return a 0 duration.
+func BiggestDurationPerTenant(tenantIDs []string, f func(string) time.Duration) time.Duration {
+	var result *time.Duration
```

Comment:
Duration is a simple number, and we already return 0 if we don't find any. I'd suggest simplifying this code to avoid pointers, and just start with `result := time.Duration(0)`

Reply:
:+1:  done that